src/Services/Tinker.php

```diff
@@ -199,7 +201,7 @@ protected function ignoreCommentsAndPhpTags(array $token)
 
     protected function cleanOutput(string $output): string
     {
-        $output = preg_replace('/(?s)(<aside.*?<\/aside>)|Exit:  Ctrl\+D/ms', '$2', $output);
+        $output = preg_replace('/(?s)(<aside.*?<\/aside>)|Exit: {2}Ctrl\+D/ms', '$2', $output);
```

Comment:
What's up this change? Not sure to understand?

Reply:
As the regex is parsing counts of 2, this assures the regex will work more accurately.